data/en/homepage.yml

```diff
@@ -5,17 +5,17 @@ slider:
   slider_item:
     # slider item loop
     - title : "#oakmtg Appreciation Club"
-      content : "A community of friends an neighbors dedicated to transparency and keeping tabs on local politics"
+      content : "A community of friends and neighbors dedicated to transparency and keeping tabs on local politics"
       animation_in : "left"
       animation_out : "right"
       # available animation : left, right, up, down
       button:
         enable : true
         label : "Learn more"
         link : "about"
-       
-       
-    - title : "The Last-Minute Voter's Guide to the Oakland Election"
+
+
+    - title : "The Informed Voter's Guide to the 2020 Oakland Election"
```

Comment:
Probably should've put this in https://github.com/astropuffin/oakmtg.club/pull/26 but I didn't find it until I was doing a spelling pass